infra/aws/terraform/prow-build-cluster/locals.tf

```diff
@@ -81,6 +82,27 @@ locals {
     }
   ]
 
+  karpenter_roles = [
+    {
+      rolearn  = module.karpenter.node_iam_role_arn
+      username = "system:node:{{EC2PrivateDNSName}}"
+      groups = [
+        "system:bootstrappers",
+        "system:nodes"
+      ]
+    }
+  ]
+
+  sso_roles = [
+    {
+      rolearn  = "arn:aws:iam::468814281478:role/AWSReservedSSO_AdministratorAccess_abaef4db15a2c055"
+      username = "sso-admins"
+      groups   = [
+        "eks-cluster-admin"
```

Comment:
not sure what permissions this group maps to, but similar to above - would cluster access entry work here instead?